browser/components/CodeEditor.js

```diff
@@ -644,6 +618,55 @@ export default class CodeEditor extends React.Component {
     }
   }
 
+  setCodeEditorLintConfig () {
```

Comment:
Hi @roottool, this should be named `getCodeEditorLintConfig` since it returns the config.